enable snapping_include_static_closure by default for reroute (#6164)
mapbox/mapbox-navigation-android: CHANGELOG.md

#### Bug fixes and improvements
- Updated `NavigationView` to allow drawing of the info panel behind the translucent navigation bar. [#6145](https://github.com/mapbox/mapbox-navigation-android/pull/6145)
- Optimized vanishing line updates (`MapboxRouteLineApi#updateTraveledRouteLine`) when going through restrictions and legs aren't styled independently. [#6169](https://github.com/mapbox/mapbox-navigation-android/pull/6169)

> - Changed `RouteOptionsUpdater` to use `snapping_include_static_closures=true` for origin of each re-route request. Similar to `snapping_include_closures=true`, but it includes statically closed roads, that is long-term. [#6164](https://github.com/mapbox/mapbox-navigation-android/pull/6164)

Reviewer: ```suggestion
- The SDK adds `snapping_include_static_closures=true` for an origin of each re-route request. [#6164](https://github.com/mapbox/mapbox-navigation-android/pull/6164)
```
I'm not sure if our users know what `RouteOptionsUpdater` is 🤔
